metadata.yaml

```diff
@@ -0,0 +1,7 @@
+homepage: "https://www.snowplowanalytics.com"
+documentation: "https://docs.snowplowanalytics.com/docs/forwarding-events-to-destinations/forwarding-events/google-tag-manager-server-side/"
```

Comment:
Might as well publish the docs and update this link before we submit it to Google.